packages/design-system-tokens/src/lib/types.ts

```diff
@@ -32,23 +32,13 @@ export type SpacerTokens = Token<MeasureValues>;
 export type TimeTokens = Token<number>;
 export type zIndexTokens = Token<number>;
 
-export type AllTokenValues =
-  | AnimationTokens
-  | BorderRadiusTokens
-  | ColorTokens
-  | FontTokens
-  | MeasureTokens
-  | MediaWidthTokens
-  | ShadowTokens
-  | SpacerTokens
-  | TimeTokens
-  | zIndexTokens;
+export type AnyTokenValues = Token<any>;
```

Comment:
What's the reason for making this less restrictive? Did a token not fit inside one of the categories?

Reply:
The places they were being used needed some more flexibility yeah, I'm not sure how much at the moment so can be revised in the future.